frontend/src/pages/Setup/IntegrationBar.tsx

```diff
@@ -122,19 +154,20 @@ export const IntegrationBar: React.FC<Props> = ({ integrationData }) => {
 const buildResourcePath = (
 	area: Area,
 	projectId: string,
-	integrationData?: IntegrationStatus,
+	resource?: Partial<Session> | Partial<ErrorGroup>,
 ) => {
-	let path = `/${projectId}/${CTA_PATH_MAP[area!]}`
+	const basePath = `/${projectId}/${CTA_PATH_MAP[area!]}`
+	let path
 
-	if (integrationData?.resourceSecureId) {
-		path = `${path}/${integrationData.resourceSecureId}`
+	if (resource?.secure_id) {
+		path = `${basePath}/${resource.secure_id}`
 	} else if (area === 'backend-logging') {
-		const logDate = moment(integrationData?.createdAt)
+		const logDate = moment(resource?.created_at)
```

Comment:
Replacing `moment` with `date-fns` could be a fun project. It would do wonders for our bundle size.

Reply:
Definitely! I think we looked at this in the past, but not sure why we didn't go through with it. Maybe @jay-khatri knows 🤔 

https://github.com/highlight/highlight/issues/3746

Reply:
That wasn't my ticket; it was just created by me in the migration 😅 . Don't have opinions here tbh; are we worried about our bundle size atm?

Reply:
No, I don't think there are any major concerns right now. Never hurts to optimize, but if we start to see more mobile / slow networks trying to use Highlight we may want to consider it (I've never actually looked to check avg connection speed or browser/OS, but assume it's mostly desktop usage).